services/autoscaling/src/simcore_service_autoscaling/dynamic_scaling_core.py

```diff
@@ -82,22 +91,19 @@ async def check_dynamic_resources(app: FastAPI) -> None:
                     InstanceTypeType, ec2_instances_needed[0].name
                 ),
                 tags={
-                    "io.osparc.autoscaling.created": f"{datetime.utcnow()}",
-                    "io.osparc.autoscaling.version": f"{VERSION}",
-                    "io.osparc.autoscaling.monitored_nodes_labels": json.dumps(
+                    "io.simcore.autoscaling.created": f"{datetime.utcnow()}",
```

Comment:
Q? why ``io.simcore`` and not ``io.osparc``. The convention is to use the reversed nmap i.e. the latter

Reply:
According to the key format recommendations https://docs.docker.com/config/labels-custom-metadata/#key-format-recommendations ... it should be the reverse DNS of a domain that **we own**, therefore I would say it should be ``io.osparc``.

![image](https://user-images.githubusercontent.com/32402063/205911878-12115ee6-49f5-411f-b00a-daede4600a2c.png)


 In any case, we have to refactor this also in other parts.  I saw you created a case for that

Reply:
I agree with you. but still since these labels are not public there is no harm done.